source/elements/oneMKL/source/domains/spblas/data_types/destroy_dense_matrix.rst

```diff
@@ -0,0 +1,55 @@
+.. SPDX-FileCopyrightText: 2024 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. _onemkl_sparse_destroy_dense_matrix:
+
+destroy_dense_matrix
```

Comment:
Are the names set on this? The routine never destroys an object, but can cause the buffer to be freed in the case the reference count for the underlying buffer (not for the matrix object) is zero.

For the sparse matrix-vector routines, there is a `release` routine, which I think is more appropriate. See, for example spmv.rst line 108. Then there would be similar naming between the matrix and matrix-vector API, which I think is cleaner

Reply:
I agree there is no reason to make the names inconsistent. I have renamed `create_*` and `destroy_*` to `init_*` and `release_*` in https://github.com/oneapi-src/oneAPI-spec/pull/522/commits/4128a65bcfd5be60fa15d8797e63d7db6fdaa008.

Note that all the `init_*` functions do create an object. They must make a host allocation to create a `handle` or a `descriptor`. They are free'd in the `release_*` functions.